websites/D/Discord Resources/dist/metadata.json

```diff
@@ -9,10 +9,10 @@
 		"en": "A list of tons of useful Discord resources and utilities for all types of users, from beginners to power users."
 	},
 	"url": "discordresources.com",
-	"version": "1.0.0",
+	"version": "1.0.1",
 	"logo": "https://i.imgur.com/LzdNPBJ.png",
 	"thumbnail": "https://i.imgur.com/LCBideV.png",
-	"color": "#1ed760",
+	"color": "#5865F2",
```

Comment:
Why?

Reply:
Because i forgot to change it (planned it all the time but forgot it). I wanted to add the official blurple instead of a green because it's fitting more

Reply:
The website is completely purple, you should use that instead

Reply:
> The website is completely purple, you should use that instead

alright, will change the color